draft-ietf-core-dns-over-coap.md

```diff
@@ -371,20 +366,22 @@ A successful response:
              1c 00 01 00 01 37 49 00 10 20 01 0d b8 00 01 00 [binary]
              00 00 01 00 02 00 03 00 04                      [binary]
 
-When a DNS error (SERVFAIL in this case) is noted in the DNS response, the CoAP
-response still indicates success:
+When a DNS error—NotImp (RCODE = 4) in response to a DNS Update (OPCODE = 5) for "example.org" in this case—is noted in the DNS response, the CoAP response still indicates success.
 
     2.05 Content
     Content-Format: application/dns-message
-    Payload: 00 00 81 a2 00 01 00 00 00 00 00 00 07 65 78 61 [binary]
-             6d 70 6c 65 03 6f 72 67 00 00 1c 00 01          [binary]
+    Payload: 00 00 a8 84 00 01 00 00 00 00 00 00 07 65 78 61 [binary]
```

Comment:
I can't easily check this change.
Is there a common pretty-printer for DNS messages that we could use?

Reply:
Asking around/Will search for one.

Reply:
There is no dedicated format in the sense of e.g. CBOR diagnostic notation (:wink:). The DNS experts I asked proposed to use some form of condensed `dig`-like formatting, as this is what everybody looks at these days.

Reply:
See 6561123